target/linux/generic/files/drivers/mtd/mtdsplit/mtdsplit_uimage.c

```diff
@@ -458,6 +458,49 @@ static struct mtd_part_parser uimage_fonfxc_parser = {
 	.parse_fn = mtdsplit_uimage_parse_fonfxc,
 };
 
+
+/**************************************************
+ * ipTIME (Kirkwood devices)
```

Comment:
I'll happily adopt `openwrt,uimage` if the [mtdsplit_uimage PR](https://patchwork.ozlabs.org/project/openwrt/cover/20201130221935.11127-1-bjorn@mork.no/) get merged.

Reply:
Finally merged! 3f36dffbdee0